CMakeLists.txt

```diff
@@ -103,12 +103,17 @@ if(DEFINED CMAKE_BUILD_TYPE)
   set_property( CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS ${CMAKE_CONFIGURATION_TYPES} )
 endif()
 
-enable_testing()
+include("cmake/HunterGate.cmake")
+HunterGate(
+    URL "https://github.com/ruslo/hunter/archive/v0.23.158.tar.gz"
```

Comment:
We can also use the `cpp-pm` release here https://github.com/cpp-pm/hunter/releases/tag/v0.23.214, so that people have access to the latest `OpenCV-Extra` release that you are adding in https://github.com/cpp-pm/hunter/pull/8

Reply:
Agree, I'll re run ci with this change to test if that breaks anything here.